source/pages/Transactions/TransactionConfirmation.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useEffect, useState } from 'react';
+
+import {
+  DefaultModal,
+  ErrorModal,
+  PrimaryButton,
+  SecondaryButton,
+} from 'components/index';
+import { useStore } from 'hooks/index';
+import { dispatchBackgroundEvent, getController } from 'utils/browser';
+import {
+  camelCaseToText,
+  capitalizeFirstLetter,
+  ellipsis,
+  formatUrl,
+} from 'utils/format';
+
+interface ITransactionConfirmation {
+  callback: any;
+  host: string;
+  title: string;
+  transaction: any;
+  type: string;
+}
+
+interface ITxData {
+  advanced: boolean;
+  label: string;
+  value: any;
+}
+
+const TransactionConfirmation: React.FC<ITransactionConfirmation> = ({
+  callback,
+  host,
+  transaction,
+  type,
+  title,
+}) => {
+  if (!transaction) throw new Error('No transaction');
+
+  const { getRecommendedFee } = getController().wallet.account.sys.tx;
+  const { activeAccount, activeNetwork } = useStore();
+
+  const [data, setData] = useState<ITxData[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+  const [submitted, setSubmitted] = useState(false);
+
+  const advancedOptionsArray = [
+    'notarydetails',
+    'notaryAddress',
+    'auxfeedetails',
+    'payoutAddress',
+    'capabilityflags',
+    'contract',
+  ];
+
+  useEffect(() => {
+    const txData: ITxData[] = [];
+    for (const [key, value] of Object.entries(transaction)) {
+      txData.push({
+        label: key,
+        value,
+        advanced: advancedOptionsArray.includes(key),
+      });
+    }
+
+    setData(txData);
+  }, [transaction]);
+
+  const onSubmit = async () => {
+    setLoading(true);
+
+    if (activeAccount.balances.syscoin <= 0) return;
```

Comment:
get balance according to the active network

Reply:
This PR is not supposed to be multi chain. This change will be implemented with the eth txs